appnav/build.gradle.kts

```diff
@@ -8,24 +8,20 @@
 @file:Suppress("UnstableApiUsage")
 
 import extension.allFeaturesApi
+import extension.setupAnvil
 
 plugins {
     id("io.element.android-compose-library")
-    alias(libs.plugins.anvil)
-    alias(libs.plugins.kapt)
     id("kotlin-parcelize")
 }
 
 android {
     namespace = "io.element.android.appnav"
 }
 
-dependencies {
-    implementation(projects.anvilannotations)
-    anvil(projects.anvilcodegen)
-    implementation(libs.dagger)
-    kapt(libs.dagger.compiler)
+setupAnvil()
```

Comment:
I think you need to pass `generateDaggerCode = true` and `generateDaggerFactoriesUsingAnvil = false` here (because `generateDaggerFactories.set(true)` was not present in the current version of the file)

Reply:
Actually, I think this was a mistake and we were generating the code twice. I removed that and set all the code generation logic in `app` and it seems to be working fine.

Reply:
OK, thanks